exercise/models.py

```diff
@@ -208,6 +208,9 @@ def __str__(self):
         help_text="Should participants receive a phishing debrief at the end of the exercise?",
         default=False,
     )
+    copied_from = models.ForeignKey(
+        "self", on_delete=models.CASCADE, null=True, blank=True
```

Comment:
Let's protect copies from being wiped by deleting the original exercise. Image if admins have copied a public exercise then someone would delete the original.
```suggestion
        "self", on_delete=models.PROTECT, null=True, blank=True
```

Reply:
Please also regenerate the migration file after this change.